netsim/ansible/templates/stp/eos.j2

```diff
@@ -36,4 +36,5 @@ interface {{ ifdata.ifname }}
 #
 {%    endif %} 
 {%   endif %}
+ spanning-tree portfast {{ ifdata.stp.port_type|default('auto') }}
```

Comment:
You should either configure this conditionally or set the default value to 'normal' (the default EOS value)

Reply:
Added 'auto' as a valid value, other platforms enable 'auto' by default. Changed the EOS template to default to 'normal'